lmdeploy/cli/utils.py

```diff
@@ -363,4 +363,9 @@ def cache_block_seq_len(parser):
             '--cache-block-seq-len',
             type=int,
             default=64,
-            help='The length of the token sequence in a k/v block')
+            help='The length of the token sequence in a k/v block. '
+            'For Turbomind Engine, if the GPU compute capability '
+            'is >= 8.0, it should be a multiple of 32, otherwise '
```

Comment:
`multiplier` would be more precise

Reply:
Maybe we can refer to [this wiki](https://en.wikipedia.org/wiki/Multiple_(mathematics)).